src/app.js

```diff
@@ -108,10 +108,46 @@ app.get("/groups/:id/jobs", function (req, res) {
       return getJobs(req.params.id, req.cookies.access_token, projects);
     })
     .then((data) => {
+      const createdCards = []
+      const pendingCards = []
+      const runningCards = []
+      const createdJobs = data.filter((data) => data.status === "created")
+      const pendingJobs = data.filter((data) => data.status === "pending")
+      const runningJobs = data.filter((data) => data.status === "running")
+      const cardTemplate = ejs.compile(
+        read("src/views/partials/singleJobCard.ejs", "utf-8")
+      );
```

Comment:
The template does not change in every request, so we should load it globally once.